README.md

```diff
@@ -99,3 +99,46 @@ $ nvidia-docker run \
     --log-opt max-file=3 \
     codalab/competitions-v2-compute-worker:nvidia 
 ```
+
+# Worker management
+
+Outside of docker containers install [Fabric](http://fabfile.org/) like so:
+
+```bash
+pip install fab-classic
+```
+
+and create a `server_config.yaml` in the root of this respository:
```

Comment:
Add a `server_config_sample.yaml` file like we have `.env_sample` ?

Reply:
added